src/sage/rings/padics/padic_printing.pyx

```diff
@@ -136,6 +137,9 @@ class pAdicPrinterDefaults(SageObject):
             sage: padic_printing.mode('digits')
             sage: repr(Qp(17)(100))
             '...5F'
+            sage: padic_printing.mode('digits-unicode')
+            sage: repr(Qp(17)(100))
+            '…0000000000000000005F'
```

Comment:
move the test right below to above `mode('digits-unicode')`. (I think the `...` is matching a `…` and that's surely unintentional)